packages/cards-editor-middleware/templates/flpSandbox.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE HTML>
+<!-- Copyright (c) 2015 SAP AG, All Rights Reserved -->
+<html lang="en">
+
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title><%- templateModel.appTitle %></title>
+	<script type="text/javascript">
+		window["sap-ushell-config"] = {
+			defaultRenderer: "fiori2",
+			bootstrapPlugins: {
+				"RuntimeAuthoringPlugin": {
+					component: "sap.ushell.plugins.rta",
+					config: {
+						validateAppVersion: false
+					}
+				}
+			},
+			renderers: {
+				fiori2: {
+					componentData: {
+						config: {
+							search: "hidden"
+						}
+					}
+				}
+			},
+			applications: {
+				"Cards-generator": {
+					title: "<%- templateModel.appTitle %>",
+					description: "",
+					additionalInformation: "SAPUI5.Component=<%- templateModel.component %>",
+					applicationType: "URL",
+					url: "../"
+				}
+			}
+		};
+	</script>
+
+    <script src="../resources/sap/ushell/bootstrap/sandbox2.js" id="sap-ushell-bootstrap"></script>
+	
+	<!-- Bootstrap the UI5 core library -->
+	<script id="sap-ui-bootstrap" src="../resources/sap-ui-core.js"
+		data-sap-ui-libs="sap.m, sap.f, sap.ushell, sap.ui.core, sap.ui.layout, sap.ui.generic.app, sap.ui.comp, sap.suite.ui.generic.template, sap.ovp, sap.ui.rta"
+		data-sap-ui-async="true" data-sap-ui-preload="async" data-sap-ui-theme="sap_horizon"
```

Comment:
You can't hard code the theme to `sap_horizon`. In case no proxy is used you must use exactly the theme that is being served by UI5 tooling. That CAN be sap_horizon ([sample](https://github.com/heimwege/cap-node-fe-ts-sample/blob/main/app/samples/ui5-local.yaml#L15)) but it can also be something else. And then the UI will not be able to render properly. See preview-middleware [template](https://github.com/SAP/open-ux-tools/blob/main/packages/preview-middleware/templates/flp/sandbox.html#L55)

Reply:
@chaudhary1-rohit do we need this change ?

Reply:
updated now